document/CoreFormat.mdk

```diff
@@ -413,25 +414,58 @@ along with its version and any other useful identifying information, such as its
 `"CodeScanner 2.0, Developer Preview (en-US)"`
 ~
 
-### `version` property { #tool-version }
+### `semanticVersion` property { #tool-semanticVersion }
 
-A `tool` object should contain a property named `version` whose value is a string containing the tool version.
-If present, the string should be in the format specified by Semantic Versioning 2.0.0.
+In a log file produced by a static analysis tool, a `tool` object shall contain a property named `semanticVersion`
+whose value is a string containing the tool version in the format specified by Semantic Versioning 2.0.0 ("SemVer").
 
-~ IsoNote
+~ IsoNumberedExample { @IsoNumberedExample=1 }
+`"2.0.0-beta.1"`
+~
+
+~ IsoNumberedNote { @IsoNumberedNote=1 }
 Semantic versions have the property of being sortable in chronological order of release.
+The presence of the `semanticVersion` property allows results management systems to (for example)
+restrict the results they display to versions newer than a specified version,
+or to restrict the results to a particular major version.
 ~
 
-~ IsoExample
-`"1.0.0-beta"`
+If the tool does not natively present its version string in SemVer format,
+it shall synthesize a SemVer string to populate the `semanticVersion` property. 
+
+~ IsoNumberedExample
+Suppose an analysis tool natively presents its version string as `"2.0"`
+(no "patch level" is available). The tool would synthesize a SemVer string `"2.0.0"`.
+~
+
+~ IsoNumberedExample
+Suppose an analysis tool natively presents its version string as `"2.0 Beta 1"`
+(the "pre-release" information is not in SemVer format).
+The tool would synthesize a SemVer string `"2.0.0-beta.1"`.
+~
+
+In a log file produced by a conversion tool, the `semanticVersion` property shall be absent.
+
+~ IsoNumberedNote
+The rationale is that a static analysis tool knows whether its version string is intended to be
+interpreted according to SemVer.
+A converter will in general not know this, even if the tool's version string conforms to the
+pattern specified by SemVer.
 ~
 
+### `version` property { #tool-version }
+
+In a log file produced by a static analysis tool, a `tool` object may contain a property named `version`
+whose value is a string containing the tool version in whatever format the tool natively provides.
+
+In a log file produced by a converter, the `version` property shall be present.
+
```

Comment:
We talked about having consumers attempt to interpret `version` as a SemVer version if `semanticVersion` was absent. But I don't think they should do that (so I didn't write that text here). If `semanticVersion` is absent, it means that the file's producer did not know whether to interpret the tool's native version string format as a semantic version. So I don't think consumers should make that guess either. I mean, they're welcome to try, but I wouldn't have the spec push them in that direction.